cypress/e2e/specs/reset_password.spec.ts

```diff
@@ -80,7 +74,7 @@ describe('Reset Password Page', () => {
 
         cy.wait('@resetPassword').then(() => {
           // Stays on the page as token is invalid
-          cy.get('[data-testid="kong-auth-error-message"]').should('contain', 'The password reset token is invalid')
+          cy.get('[data-testid="kong-auth-error-message"]').should('contain', 'An internal failure occurred')
```

Comment:
This message is what's coming from KAuth - I think if we want a better message, it should be handled at KAuth's level since the API _could_ theoretically throw a 500.

Reply:
cc @kinman

Reply:
Thanks @nateslo . This is a regression - the spec doesn't say we throw a 500 in this case, and we shouldn't. I'll make sure we have a test case for this.